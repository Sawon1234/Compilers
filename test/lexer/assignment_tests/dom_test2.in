5 != 7; 

10 > "I am a string!"

5 + 7 = "foo" - bar;
[6] ++ [10,9,8]

