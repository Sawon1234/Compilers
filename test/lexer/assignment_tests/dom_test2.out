0 != 0 ; 0 > "" 0 + 0 = "" - name ; [ 0 ] ++ [ 0 , 0 , 0 ]
